stripper/ze_chicken_lords_p.cfg

```diff
@@ -0,0 +1,16 @@
+;tp angles
+modify:
+{
+	match:
+	{
+		"classname" "trigger_teleport"
+	}
+	replace:
```

Comment:
The replace is unnecessary, as there are no existing UseLandmarkAngles keyvalues with a value not already of 1.